T04_munsakad_sherrif.py

```diff
@@ -0,0 +1,23 @@
+from time import sleep
+delay = 2
+direction = input("Go with him further [Up, or Down]?")
+if direction == "Up":
+    print("You've reached a large vault... a large console stands to its right...")
+    print("The console lights up when you reach it... it asks for a passcode")
+code = input("Guess the code/Ask for help")
+if code == "Ask for help":
+    print("You ask Dr. Heggen for help")
+    print("He pulls out a small electrical device and plugs it into the console")
+    print("The console glows green and the vault opens up")
+    print("you've found riches beyond your wildest dreams")
```

Comment:
Any path leading to the user staying alive should end with `return True`. Any path that kills them should `return False`.